plugins/woocommerce/tests/e2e-pw/tests/shopper/shop-products-filter-by-price.spec.js

```diff
@@ -24,7 +24,14 @@ const test = baseTest.extend( {
 
 test.describe(
 	'Filter items in the shop by product price',
-	{ tag: [ '@payments', '@services' ] },
+	{
+		tag: [
+			'@payments',
+			'@services',
+			'@skip-on-default-wpcom',
+			'@skip-on-default-pressable',
```

Comment:
Again, in this test there is wp-env CLI execution to check current WordPress version. Skipping until we figure out how to execute separate CLI commands on external vs wp-env.